.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
sorry, global gitignore. had npmignores on the mind. This stuff should be in a global gitignore.